src/config.js

```diff
@@ -23,7 +23,8 @@ const config = {
     "citations-and-warnings":{
       chartId: "chart-id"
     }
-  }
+  },
+  sandbox: false,
```

Comment:
Took a little creative liberty with naming this - open to other names!
The flag should be set as true when enabling the feature.